### DISCUSSION TITLE
bulk import python script doesn't work

### DISCUSSION THREAD

**Discussion by BrandonG777** [mealie-recipes/mealie#441] [Q&A]
Getting the following error...

<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="icon" href="/favicon.ico"><title>Mealie</title><link href="/css/app.5828cb39.css" rel="preload" as="style"><link href="/css/chunk-vendors.f6d68ce2.css" rel="preload" as="style"><link href="/js/app.22e1d730.js" rel="preload" as="script"><link href="/js/chunk-vendors.37a4569f.js" rel="preload" as="script"><link href="/css/chunk-vendors.f6d68ce2.css" rel="stylesheet"><link href="/css/app.5828cb39.css" rel="stylesheet"></head><body><noscript><strong>We're sorry but frontend doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id="app"></div><script src="/js/chunk-vendors.37a4569f.js"></script><script src="/js/app.22e1d730.js"></script></body></html>

UPDATE: Got the bash script to work. I'm a bit more familiar with bash scripts, so it was easier for me to troubleshoot that one.

**Comment by tehniemer** [comment]
I'm having problems with the bash script, how did you get it working?
